src/database.rs

```diff
@@ -240,9 +241,10 @@ pub async fn get_floppa(ctx: &Context, n: Option<u32>) -> Option<String> {
         )
         .await
         .ok()?;
-
-    let floppa_id = if n.is_some() && ids.contains(&n.unwrap()) {
-        n.unwrap()
+    
+    
+    let floppa_id = if n.is_some() && ids.len() > 0 { 
+        *ids.get(cmp::max(0, n.unwrap() as usize % ids.len())).unwrap()

```

Comment:
floppa ids actually start at 1

Reply:
That doesn't matter. It's the index of an id, not the id itself.